client/src/research/BubbleChart.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {percentRightPerProfile} from './calculations';
+import _ from 'lodash';
+import {
+  isSwipe
+} from './functions.js';
+
+class BubbleChart extends Component{
```

Comment:
Please add a short comment (1-2 lines) describing what this component is for and what it does.